homeassistant/components/ruckus_unleashed/__init__.py

```diff
@@ -68,11 +79,12 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
 async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Unload a config entry."""
+
     unload_ok = await hass.config_entries.async_unload_platforms(entry, PLATFORMS)
     if unload_ok:
         for listener in hass.data[DOMAIN][entry.entry_id][UNDO_UPDATE_LISTENERS]:
             listener()
-
+            await hass.data[DOMAIN][entry.entry_id][COORDINATOR].ruckus.close()
```

Comment:
This shouldn't be indented in the for loop.

Reply:
Fixed in https://github.com/home-assistant/core/pull/99411